modules/integration/tests-integration/tests-es/src/test/java/org/wso2/carbon/registry/es/notes/GregCustomRxtNotesTestESTestCase.java

```diff
@@ -0,0 +1,217 @@
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Factory;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.automation.engine.frameworkutils.FrameworkPathUtil;
+import org.wso2.carbon.integration.common.utils.mgt.ServerConfigurationManager;
+import org.wso2.carbon.registry.core.exceptions.RegistryException;
+import org.wso2.carbon.registry.es.utils.ESTestCommonUtils;
+import org.wso2.carbon.registry.es.utils.GregESTestBaseTest;
+import org.wso2.carbon.registry.resource.stub.ResourceAdminServiceExceptionException;
+import org.wso2.greg.integration.common.clients.ResourceAdminServiceClient;
+import org.wso2.greg.integration.common.utils.GenericRestClient;
+
+import javax.activation.DataHandler;
+import javax.ws.rs.core.MediaType;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GregCustomRxtNotesTestESTestCase extends GregESTestBaseTest {
+    private static final Log log = LogFactory.getLog(GregCustomRxtNotesTestESTestCase.class);
+    private TestUserMode userMode;
+    String jSessionId;
+    String assetId;
+    String assetType;
+    String cookieHeaderPublisher;
+    GenericRestClient genericRestClient;
+    Map<String, String> queryParamMap;
+    Map<String, String> headerMap;
+    String publisherUrl;
+    String resourcePath;
+    String noteName;
+    String noteOverviewHash;
+    String noteAssetId;
+    String replyAssetId;
+    ESTestCommonUtils esTestCommonUtils;
+    private ResourceAdminServiceClient resourceAdminServiceClient;
+
+    @Factory(dataProvider = "userModeProvider")
+    public GregCustomRxtNotesTestESTestCase(TestUserMode userMode) {
+        this.userMode = userMode;
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+        super.init(userMode);
+        String session = getSessionCookie();
+        noteName = "testNote33";
+        assetType = "wsdl";
+        genericRestClient = new GenericRestClient();
+        queryParamMap = new HashMap<String, String>();
+        headerMap = new HashMap<String, String>();
+        resourcePath =
+                FrameworkPathUtil.getSystemResourceLocation() + "artifacts" + File.separator + "GREG" + File.separator
+                        + "json" + File.separator;
+        publisherUrl = automationContext.getContextUrls().getSecureServiceUrl().replace("services", "publisher/apis");
+        esTestCommonUtils = new ESTestCommonUtils(genericRestClient, publisherUrl, headerMap);
+        resourceAdminServiceClient = new ResourceAdminServiceClient(backendURL, session);
+        addCustomRxt();
+        new ServerConfigurationManager(automationContext).restartGracefully();
+        Thread.sleep(120000);
+        SetTestEnvironment();
```

Comment:
Thanks for pointing it out. Fixed in the following commit - https://github.com/wso2/product-greg/pull/274/commits/470633fc39549bf01b238abf015cd6d5abdc11bb